doc/introduction/index.rst

```diff
@@ -411,448 +391,37 @@ The Boot process proceeds as follows:
    the ACRN Device Model and Virtual bootloader through ``dm-verity``.
 #. The virtual bootloader starts the User-side verified boot process.
 
-In this boot mode, the boot options of pre-launched VM and service VM are defined
+In this boot mode, the boot options of a pre-launched VM and the Service VM are defined
 in the variable of ``bootargs`` of struct ``vm_configs[vm id].os_config``
 in the source code ``configs/scenarios/$(SCENARIO)/vm_configurations.c`` (which
 resides under the hypervisor build directory) by default.
-Their boot options can be overridden by the GRUB menu. See :ref:`using_grub` for
+These boot options can be overridden by the GRUB menu. See :ref:`using_grub` for
 details. The boot options of a post-launched VM are not covered by hypervisor
 source code or a GRUB menu; they are defined in a guest image file or specified by
```

Comment:
Although we are not changing "guest" widely for the 2.6 release, I suggest changing it in this one case for clarity. Suggestion:
```suggestion
source code or a GRUB menu; they are defined in the VM's OS image file or specified by
```